tests/contrib/litellm/conftest.py

```diff
@@ -1,34 +1,75 @@
-import pytest
+import mock
 
+import pytest
 from ddtrace.contrib.internal.litellm.patch import patch
-from ddtrace.contrib.internal.litellm.patch import unpatch
 from ddtrace.trace import Pin
-from tests.contrib.litellm.utils import get_request_vcr
+from ddtrace.contrib.internal.litellm.patch import unpatch
 from tests.utils import DummyTracer
 from tests.utils import DummyWriter
+from tests.utils import override_global_config
+from tests.contrib.litellm.utils import get_request_vcr
+from ddtrace.llmobs import LLMObs
+
+
+def default_global_config():
+    return {}
 
 
 @pytest.fixture
-def litellm(monkeypatch):
-    monkeypatch.setenv("OPENAI_API_KEY", "<not-a-real-key>")
-    monkeypatch.setenv("ANTHROPIC_API_KEY", "<not-a-real-key>")
-    monkeypatch.setenv("COHERE_API_KEY", "<not-a-real-key>")
-    patch()
-    import litellm
+def ddtrace_global_config():
+    return {}
 
-    yield litellm
-    unpatch()
+
+@pytest.fixture()
+def mock_llmobs_writer():
+    patcher = mock.patch("ddtrace.llmobs._llmobs.LLMObsSpanWriter")
+    try:
+        LLMObsSpanWriterMock = patcher.start()
+        m = mock.MagicMock()
+        LLMObsSpanWriterMock.return_value = m
+        yield m
+    finally:
+        patcher.stop()
```

Comment:
Can we use the new test fixture instead of mocking llmobs writer? This fixture has flaked and we're now switching over to use events/writer fixtures:
- https://github.com/DataDog/dd-trace-py/blob/main/tests/contrib/botocore/test_bedrock_llmobs.py#L108-L111
- https://github.com/DataDog/dd-trace-py/blob/main/tests/contrib/botocore/test_bedrock_llmobs.py#L130-L144